### PR TITLE
Added mousetrap

### PR DESCRIPTION
**Pull request by tkroll**
Switched to non-minified Mousetrap

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "manifest_version": 2,
     "name": "Google Search Navigator",
-    "version": "0.2.11",
+    "version": "0.2.91",
```
Comment (infokiller) [inline src/manifest.json]: 0.2.12

Comment (tkroll) [reply src/manifest.json]: Done

**Comment by infokiller** [comment]
@tkroll just a heads up I plan to migrated the code to typescript. Hope it doesn't make merging/rebasing this a big issue, but let me know if I can help somehow. Just don't want to create extra work for you if I can avoid it :)

**Comment by tkroll** [comment]
Typescript is fine for me. Will the external mousetrap.js play nicely with your build?

**Comment by infokiller** [comment]
Yes, it should play nicely, and also provide good autocompletion and other benefits of using typescript, since it has type definitions: https://www.npmjs.com/package/@types/mousetrap

**Comment by tkroll** [comment]
Fork was outdated. Created new one

**Pull request closed without merging**